### PR TITLE
Update fields and fix number templating bug

### PR DESCRIPTION
**Pull request by jaserud**
changes:
- add `diagnostic_pcr_ct_value_null_reason` and `GISAID accession` to tsv schema and payload template
- fix Number checking and templating bug
- update tests as needed

### CODE REVIEW

File: src/main/resources/tsv-schema.json [hunk 1]
```diff
@@ -33,11 +33,13 @@
   { "name": "raw sequence data processing method", "valueType": "string" },
   { "name": "dehosting method", "valueType": "string" },
   { "name": "consensus sequence software name", "valueType": "string" },
-  { "name": "consensus sequence software version", "valueType": "number" },
+  { "name": "consensus sequence software version", "valueType": "string" },
```
Comment (jaserud) [inline src/main/resources/tsv-schema.json:36]: Bug was discovered when this was changed to string. The previous approach was converting string that are numbers to numbers, so setting this to string wasn't working.

File: src/main/resources/tsv-schema.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   { "name": "collection method", "valueType": "string" },
   { "name": "host (scientific name)", "valueType": "string" },
   { "name": "host disease", "valueType": "string" },
-  { "name": "host age", "valueType": "string" },
+  { "name": "host age", "valueType": "number" },
```
Comment (jaserud) [inline src/main/resources/tsv-schema.json:24]: This was found after the bug was fixed. This is supposed to be number, but the bug never surfaced it.

File: src/main/java/org/cancogenvirusseq/muse/components/PayloadFileMapper.java [hunk 3]
```diff
@@ -142,16 +144,16 @@ public PayloadFileMapper(String payloadJsonTemplate) {
...
   private static ObjectNode convertRecordToPayload(
-      Map<String, String> valuesMap, String payloadTemplate) {
+      Map<String, Object> valuesMap, String payloadTemplate) {
     StringLookup lookupFunc =
         key -> {
           val value = valuesMap.getOrDefault(key, "");
 
           // value is mapped to json value by these rules
-          if (NumberUtils.isCreatable(value)) {
+          if (value instanceof Number) {
```
Comment (jaserud) [inline src/main/java/org/cancogenvirusseq/muse/components/PayloadFileMapper.java:150]: this is the main crux of the bug. Changed the records to be Map<String, Object> and here we check if the type of Object is Number rather then seeing if the string is a number.

**Review by lepsalex — APPROVED** [review]
LGTM :+1:

**Pull request merged**